fdbserver/workloads/BlobRestoreWorkload.actor.cpp

```diff
@@ -120,6 +119,16 @@ struct BlobRestoreWorkload : TestWorkload {
 		return targetVersion;
 	}
 
+	static Future<Void> disableManifestBackup(Database cx) {
+		return runRYWTransaction(cx, [](Reference<ReadYourWritesTransaction> tr) -> Future<Void> {
+			tr->setOption(FDBTransactionOptions::ACCESS_SYSTEM_KEYS);
+			tr->setOption(FDBTransactionOptions::LOCK_AWARE);
+			tr->setOption(FDBTransactionOptions::PRIORITY_SYSTEM_IMMEDIATE);
+			BlobGranuleBackupConfig().enabled().set(tr, false);
+			return Void();
+		});
+	}
+
```

Comment:
Similar to above, this simplies to

```suggestion
	static Future<Void> disableManifestBackup(Database cx) {
		return BlobGranuleBackupConfig().enabled().set(SystemDBWriteLockedNow(cx.getReference()), false);
	}
```